srcalc/src/main/java/gov/va/med/srcalc/service/DefaultAdminService.java

```diff
@@ -104,7 +104,47 @@ public void saveVariable(final AbstractVariable variable)
     @Transactional(readOnly = true)
     public ImmutableCollection<RiskModel> getAllRiskModels()
     {
-        fLogger.debug("Getting all RiskModels.");
+//        fLogger.debug("Getting all RiskModels.");
         return fRiskModelDao.getAllRiskModels();
     }
+    
+    @Override
+    @Transactional(readOnly = true)
+    public RiskModel getRiskModelForId(final int modelId)
+    {
+//        fLogger.debug("Getting RiskModel {}.", modelId);
+        
+        final ImmutableCollection<RiskModel> models = getAllRiskModels();
+        for( RiskModel rm : models ) 
+        {
+        	if( rm.getId() == modelId ) {
+        		return rm;
+        	}
+        }
+        
+        fLogger.warn("Unable to find RiskModel for ID {}.", modelId);
+
+        return null;
+    }
+
+    /**
+     * Saves the name of the {@link RiskModel} 
```

Comment:
This Javadoc is inaccurate: this method actually saves the whole risk model, including its terms.